### PR TITLE
chore: update version of generate pdf package from schema tools

### PR DESCRIPTION
**Pull request by eamon0989**
`v0.0.5` uses an old version of `@s1seven/schema-tools-generate-pdf`, this PR updates it to the latest version, adding support for newer schema versions.
Add tests for `generate pdf node`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
 	"node-red": {
 		"nodes": {
 			"generate pdf": "generate-pdf-node.js"
 		}
 	},
 	"dependencies": {
-		"@s1seven/schema-tools-generate-pdf": "^0.1.1",
+		"@s1seven/schema-tools-generate-pdf": "^0.2.3",
 		"lato-font": "^3.0.0",
 		"lodash": "^4.17.21"
```
Comment (getlarge) [inline package.json]: do you really need lodash ?
Also can reorder the keys so that dependencies comes right before devDependencies ?

**Review by getlarge — CHANGES_REQUESTED** [review]
Good to have a test suite. 
I'll contain myself to comment your choice of indentation, but i just did :)

**Comment by eamon0989** [comment]
> Good to have a test suite. I'll contain myself to comment your choice of indentation, but i just did :)

Bad habit, I assumed prettier had taken care of it, guess that's another thing to add 😁

File: package.json (hunk 1 above)
Comment (eamon0989) [reply package.json]: I don't use `lodash` explicitly, but for some reason `@s1seven/schema-tools-generate-pdf` doesn't resolve the dependency. I'll see if I can find a solution.

As for ordering the keys, done 👍🏼

Comment (eamon0989) [reply package.json]: I tried removing `lodash` but I couldn't get it to resolve the dependency from `@s1seven/schema-tools-generate-pdf`. I've added it back in for now.

**Review by getlarge — APPROVED** [review]

**Pull request merged**